src/Spec.ts

```diff
@@ -1137,6 +1137,11 @@ ${await utils.readFile(path.join(__dirname, '../js/multipage.js'))}
     const location = this.opts.location;
     const stage = this.opts.stage;
 
+    const shortcutsHelp = this.doc.createElement('div');
+    shortcutsHelp.setAttribute('id', 'shortcuts-help');
+    shortcutsHelp.innerHTML = getBoilerplate('shortcuts-help');
```

Comment:
Ideally we'd only mention `m` when `this.opts.multipage` is true (it doesn't do anything otherwise). So probably just inline the boilerplate into a template?

```suggestion
    shortcutsHelp.innerHTML = `
<ul>
  <li><span>Toggle shortcuts help</span><code>?</code></li>
  <li><span>Toggle "can call user code" annotations</span><code>u</code></li>
${this.opts.multipage ? `<li><span>Navigate to/from multipage</span><code>m</code></li>` : ''}
  <li><span>Jump to search box</span><code>/</code></li>
</ul>`;
```

Reply:
Done.